app/views/login/index.html.erb

```diff
@@ -0,0 +1 @@
+<h1> Login </h1>
```

Comment:
This will get merged over eventually with the session new page but that is something to do later.  Will added to the to do list